docs/data-protection-policy.md

```diff
@@ -0,0 +1,26 @@
+# Data Protection Policy
```

Comment:
Could you update this with the google doc we've created, @victor-yunenko?

Reply:
@victor-yunenko, I've updated this with our google doc. Could you double check whether this is OK as it is now?